projects/ui/src/components/Analytics/Bean/index.tsx

```diff
@@ -9,12 +9,12 @@ import Volume from '~/components/Analytics/Bean/Volume';
 import Liquidity from '~/components/Analytics/Bean/Liquidity';
 import Crosses from '~/components/Analytics/Bean/Crosses';
 import DeltaB from '~/components/Analytics/Bean/DeltaB';
-
-// const SLUGS = ['price', 'volume', 'liquidity', 'mktcap', 'supply', 'crosses'];
 import { FC } from '~/types';
 
+const SLUGS = ['price', 'volume', 'liquidity', 'mktcap', 'supply', 'crosses', 'DeltaB'];
+
 const BeanAnalytics: FC<{}> = () => {
-  const [tab, handleChangeTab] = useTabs();
+  const [tab, handleChangeTab] = useTabs(SLUGS, 'lens');
```

Comment:
This tab refers to analytics about the Bean token, would suggest using `bean` instead of `lens`